src/themes/bankrate/theme.json

```diff
@@ -23,7 +23,9 @@
   },
   "lineHeights": {
     "base": "1.5",
-    "heading": "1.3"
+    "heading": "1.3",
+    "sm": "16px",
```

Comment:
It's better to use the spacing variables defined in the top of the file rather than hard-coded values

Reply:
I know, but design requires 16px, and Chris Underdown explicitly said it is required, 1.3 turns out to be more than 16px

Reply:
All good